lib/transmissions.js

```diff
@@ -81,3 +91,97 @@ module.exports = function(client) {
   };
 
 };
+
+function formatPayload(transmission) {
+  // format all the original recipients to be in the object format
+  transmission.recipients = _.map(transmission.recipients, (recipient) => {
+    recipient.address = addressToObject(recipient.address);
+
+    return recipient;
+  });
+
+  // add the CC headers
+  if (_.has(transmission, 'cc') && _.isArray(transmission.cc)) {
+    addCCHeader(transmission);
+  }
+
+  addListToRecipients(transmission, 'cc');
+  addListToRecipients(transmission, 'bcc');
```

Comment:
For these `add*` methods (`addCCHeader` and `addListToRecipients`) I think I'd usually prefer to see them return values that get added to the modified object here, instead of modifying/mutating in place. That'd mean we'd probably need to do the `delete` separately, here, but that might be worth it? Do you have thoughts on mutation lol